novuhq/website
# feat: add the Terms of Use and Privacy Policy pages

**Describe what changes this pull request brings**
This pull request adds the `Terms of Use` and `Privacy Policy` pages.


**Steps to test**
1. Open [the Terms of Use page](https://websitemain13551-staticpages.gtsb.io/terms/)
2. Open [the Privacy Policy page](https://websitemain13551-staticpages.gtsb.io/privacy/)
3. Confirm that everything looks and works as expected
